.github/workflows/algolia-indexing.yml

```diff
@@ -0,0 +1,32 @@
+name: Upload Data to Algolia Index
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - v1
+      - v2
+jobs:
+  upload_data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true  # Fetch Hugo themes (true OR recursive)
+          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+      - name: Setup Node.js 16.15.x
```

Comment:
@robwalsh21 Why does the version need to be this specific?

Reply:
@stoobie it can be less specific like 16. I did not try other versions. Do we have a preferred version? I can set it and try.